src/status_im/ui/components/chat_icon/styles.cljs

```diff
@@ -8,7 +9,9 @@
    :align-items      :center
    :justify-content  :center
    :border-radius    20
-   :background-color color})
+   :background-color color
+   :border-width     0.5
```

Comment:
Yes, you are right, they should not be changed. If there is no emoji, then definitely it will also have a different color set, So I can stop worrying about what will happen if bg is white. I will remove these changes. Thanks